### DISCUSSION TITLE
Activity tracker is not tracking  nes games properly

### DISCUSSION THREAD

**Discussion by realmrcool** [OnionUI/Onion#1696] [Support]
So the activity tracker is working, but not for NES emulation:
1) It doesn't count each session as a play for the same ROM but starts a new entry every time i start a session.  This is exclusively happening with nes games.
2) It tracked a NES game I had on standby for 2 hours, although I never played it; I just booted it up and put the mini in hibernate.

**Comment by Aemiii91** [comment]
Can you attach any specific examples?

**Reply by realmrcool** [reply]
Do you  have had  any time jetzt to look  into it?

**Comment by realmrcool** [comment]
https://github.com/user-attachments/assets/26b34749-618f-4e59-a2b6-2040ac65bb32

Here you can see it in action: every session is a new entry. Plus I haven't played 2 hours of bible games....

**Reply by Aemiii91** [reply]
I have no idea how that could even happen. Can you attach your play activity db? (I can't take a look right now, but will as soon as I can)

It's located at: `Saves/CurrentProfile/play_activity/play_activity_db.sqlite`

**Reply by realmrcool** [reply]
thank you for taking the time :)
i named it a .txt file since it needed a file-ending for uploading
[play_activity_db.sqlite.txt](https://github.com/user-attachments/files/17562877/play_activity_db.sqlite.txt)
